src/openwith.js

```diff
@@ -16,21 +16,31 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 OME.setOpenWithUrlProvider("omero_iviewer", function(selected, url) {
+    selected_count = selected.length;
+    // last selected image
+    last_selected = selected[selected_count-1];
     // Add image Id to url
-    url += selected[0].id + "/";
+    url += last_selected.id + "/";
```

Comment:
I think the natural order should be preserved and the initial image shown should be the first image, which is also the first in the list of thumbnails as suggested by @pwalczysko above.